x-pack/test/apm_api_integration/tests/metrics_charts/serverless.spec.ts

```diff
@@ -0,0 +1,244 @@
+            'Compute usage',
+            'System memory usage',
+          ]);
+        });
+
+        it('returns correct overallValue on avg duration chart', () => {
+          const metric = metrics.charts.find((chart) => chart.key === 'avg_duration');
+          expect(metric).not.to.be.empty();
+          const billedDurationSeries = metric?.series.find(
+            ({ title }) => title === 'Billed Duration'
+          );
+          expect(billedDurationSeries).not.to.be.empty();
+          expect(billedDurationSeries?.overallValue).to.equal(BILLED_DURATION_MS);
+
+          const transactionDurationSeries = metric?.series.find(
+            ({ title }) => title === 'Transaction Duration'
+          );
+          expect(transactionDurationSeries).not.to.be.empty();
+          expect(transactionDurationSeries?.overallValue).to.equal(TRANSACTION_DURATION * 1000);
+        });
+
+        it('returns correct overallValue on Cold start chart', () => {
+          const metric = metrics.charts.find((chart) => chart.key === 'cold_start_duration');
+          expect(metric).not.to.be.empty();
+          expect(metric?.series[0].overallValue).to.equal(COLD_START_DURATION_PYTHON);
+        });
+
+        it('returns correct overallValue on System memory usage chart', () => {
+          const metric = metrics.charts.find((chart) => chart.key === 'memory_usage_chart');
+          expect(metric).not.to.be.empty();
+
+          const memoryValue = roundNumber(1 - MEMORY_FREE / MEMORY_TOTAL);
+
+          const maxMemorySeries = metric?.series.find(({ title }) => title === 'Max');
+
+          expect(maxMemorySeries).not.to.be.empty();
+          expect(roundNumber(maxMemorySeries?.overallValue)).to.equal(memoryValue);
+
+          const avgMemorySeries = metric?.series.find(({ title }) => title === 'Average');
+          expect(avgMemorySeries).not.to.be.empty();
+          expect(roundNumber(avgMemorySeries?.overallValue)).to.equal(memoryValue);
+        });
+
+        it('returns correct overallValue on Compute usage chart', () => {
+          const metric = metrics.charts.find((chart) => chart.key === 'compute_usage');
+          expect(metric).not.to.be.empty();
+          const bytesMs = MEMORY_TOTAL * BILLED_DURATION_MS;
+          const gbSecs = bytesMs / (1024 * 1024 * 1024 * 1000);
+          expect(metric?.series[0].overallValue).to.equal(gbSecs);
+        });
+
+        it('returns correct overallValue on Active instances chart', () => {
+          const metric = metrics.charts.find((chart) => chart.key === 'active_instances');
+          expect(metric).not.to.be.empty();
+          expect(metric?.series[0].overallValue).to.equal(1);
+        });
+
+        it('returns correct overallValue on cold start count chart', () => {
+          const metric = metrics.charts.find((chart) => chart.key === 'cold_start_count');
+          expect(metric).not.to.be.empty();
+          expect(metric?.series[0].overallValue).to.equal(30);
```

Comment:
where is this number coming from?

Reply:
fixed it here https://github.com/elastic/kibana/pull/139041/commits/cfe36ac632b3fa1a6e1b54868fafa981e14df8dd#diff-c4258177e36431d334f358c863471cc966bc383bf0816383ca2abddee3810123R174